### ISSUE TITLE
[Block] File: Embedded PDF fails to render 

### ISSUE THREAD

**Issue by cuemarie** [WordPress/gutenberg#55493]
### Description

Embedded PDF files fail to render in Safari Version 17.0 when using Gutenberg 16.7.0 and higher. GB 16.6.0 works as expected.

### Step-by-step reproduction instructions

1. Site with Gutenberg 16.7.0+ active
2. Add the File Block to a page or post and embed a PDF
3. Enable "Show Inline Embed"
4. Publish the page
5. View the page in a Safari browser

### Screenshots, screen recording, code snippet

Comparing GB 16.6.0 and GB 16.8.1 in Firefox (left) and Safari (right)

https://github.com/WordPress/gutenberg/assets/27249804/169f0ac9-4efa-4084-becf-a0c174e7ac71

**GB 16.6.0**

![Markup on 2023-10-19 at 11:26:36](https://github.com/WordPress/gutenberg/assets/27249804/ed8022be-f77d-4bff-b1fd-171669ecd561)

**GB 16.7.0 and up**

![Markup on 2023-10-19 at 11:27:52](https://github.com/WordPress/gutenberg/assets/27249804/7996169a-f49a-455a-8be9-07c2200ca753)

**Safari failure to reload**

I also find if I leave the page long enough, the page fails to reload, and I have trouble viewing anything in the browser console:

<img width="550" alt="screenshot" src="https://github.com/WordPress/gutenberg/assets/27249804/c2756869-dbc2-44f3-bb8e-f678cac3973c">

### Environment info

- WordPress 6.3.2
- Gutenberg 16.8.1, 16.8.0, 16.7.1, 16.7.0
- Theme in tests: Twenty Twenty-Three
- PDF file I used: https://www.africau.edu/images/default/sample.pdf

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by apeatling** [comment]
Confirmed this to be an issue in trunk with Safari latest. I noticed that the first render of the PDF in the editor works correctly, but if you toggle the inline embed setting it breaks. It did not work in the front end.

<img width="1203" alt="Screenshot 2023-10-25 at 11 03 32 AM" src="https://github.com/WordPress/gutenberg/assets/1464705/dd3990d0-7014-4c85-9fa8-e2197237f7fb">

**Comment by ironprogrammer** [comment]
## Test Report
I am able to reproduce this issue. Removing `Needs Testing`. Upgrading from GB 16.6.0 to 16.7.0+ presents this issue.

Also noteworthy: With GB deactivated, WP 6.4-beta/RC and `wordpress-develop:trunk` also exhibit this issue. Upgrading from WP <= 6.3 to WP 6.4-beta/RC presents this issue on existing posts with File block.

### Environment
- Hardware: MacBook Pro Apple M1 Pro
- OS: macOS 13.6
- Browser: Safari 17.0, Google Chrome 118.0.5993.88, Mozilla Firefox 118.0.2
- Server: nginx/1.25.2
- PHP: 8.2.11
- WordPress: 6.5-alpha-56966-src, 6.3.1
- Theme: twentytwentythree v1.2, twentytwentyfour v1.0
- Active Plugins:
  - gutenberg v16.9.0-rc.1, v16.6.0, v16.7.0, v16.8.1

### Actual Results
- ✅ PDF is not displayed correctly on frontend with GB 16.7.0+ or `trunk`. Observed the following permutations of this issue:
  - Page loads with gray box in place of the PDF.
  - Page displays with blank area in place of PDF, and eventually times out with error from issue description.
  - Page does not display anything, and eventually times out with error.

**Comment by tomandersen** [comment]
Is this happening again? I have this exact problem on Version 17.4.1 Safari desktop, and WordPress 6.5.3. Only on safari  desktop, and toggling the PDF "Show inline embed  " OFF fixes the problem.

**Comment by ironprogrammer** [comment]
I've since updated to Safari 17.5 and cannot reproduce the frontend inline embed issue (testing 6.5.3/trunk with GB on/off).

HOWEVER, there is still a rendering issue in the editor after toggling "Show inline embed" **off and back on** (as noted in https://github.com/WordPress/gutenberg/issues/55493#issuecomment-1778829000 above). Since the originally reported issue and [patch](https://github.com/WordPress/gutenberg/pull/55667) focused on the frontend, I've opened a separate ticket that focuses on the backend behavior, https://github.com/WordPress/gutenberg/issues/62120.

**Comment by annezazu** [comment]
Reopening this as I cannot get a PDF to show a preview in both chrome and safari using WP 6.6.1 and Gutenberg 19.1. I'm using a simple WordPress playground setup and it just shows as an empty frame both in the editor and on the front end:
![Screenshot 2024-09-10 at 3 28 39 PM](https://github.com/user-attachments/assets/6b9b2117-b1f6-475b-8897-596227b3f9ec)

**Comment by ironprogrammer** [comment]
Hi, @annezazu 👋🏻 I confirmed that in Playground, the embed isn't showing up as you've reported. This looks like that might be related to https://github.com/WordPress/wordpress-playground/issues/1314, where the iframe with the embed is blocked by Chrome and Safari.

However, for a fresh local install of 6.6.2 w/ and w/o Gutenberg, the PDF does appear for me in the editor and front end:

<img alt="demo of pdf embed on fresh local 6.6.2 install" src="https://github.com/user-attachments/assets/efb3b57e-4237-46c4-bcae-daf032a1cd1b" width="50%">

Should this be closed back up?

**Comment by annezazu** [comment]
Amazing sleuthing. Foiled again by Playground. Yes, closing!
